### PR TITLE
stylized melee weapons to make it easy to read to update Pigeon's cha…

### PR DESCRIPTION
**Pull request by bleehu**
…racter

### CODE REVIEW

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt [hunk 1]
```diff
@@ -6,225 +6,286 @@ a 'Miss Chance'. This modifier is added to your combat dex modifier and
...
-Range:        	    Adjacent (1m)
+==Hand-to-Hand [untrained] Free==
+
+Damage:    5 + STR + DEX
+Range:            Adjacent (1m)
 Accuracy Modifier:  -2
 DEX Requirement:    2
 
-----Hand-to-Hand [Trained]	----
+==Hand-to-Hand [Trained]==
```
Comment (Turtlelord26) [inline CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt]: Unambiguously could use a blank line after this subsection heading for readability.

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt [hunk 2]
```diff
@@ -6,225 +6,286 @@ a 'Miss Chance'. This modifier is added to your combat dex modifier and
...
 (10 skill in Melee Combat or Unarmed Combat)
```
Comment (Turtlelord26) [inline CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt]: How will this line parse?
Maybe could remove the () and give it a "Prerequisites: " to join it to the rest of the definition list.

**Review by Turtlelord26 — CHANGES_REQUESTED** [review]
Many section and subsection headers missing whitespace, not sure if parser handles gracefully.

One more note about blanks after chapters but at this point I may be fighting a losing battle.

I would make Unarmed Fighting and Pistols into chapter headings. Particularly for pistols I see consecutive section headings as questionable style.

Inconsistent spacing in many locations in Definition lists, including lack of space after the colon in some cases. After seeing edits to other weapon pages standardizing those I will bring it up here.

Some grenades have not had their descriptions line length limited.

**Comment by bleehu** [comment]
Great catches! Thanks for the review! 

What do you think of this?

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt [hunk 3]
```diff
@@ -6,225 +6,287 @@ a 'Miss Chance'. This modifier is added to your combat dex modifier and
...
-(Requires Combat Expertise [Melee] and either 20 skill
-in Melee Combat or Unarmed Combat, or class passive.)
-Damage:		   [3*STR or 2*DEX] + [Melee Combat skill]
-Range:     	   2m
+DEX Requirement: 3
+
+	Round down damage. 20% chance to disarm target unless weapon is attached via 
+sling, part of the target, etc.
+
+==Hand-to-Hand [Expert]==
```
Comment (Turtlelord26) [inline CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt]: Mark

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt [hunk 4]
```diff
@@ -6,225 +6,287 @@ a 'Miss Chance'. This modifier is added to your combat dex modifier and
...
+
+	50% chance to disarm target (unless weapon is part of target). Can use 
+disarm to grapple an opponent using their weapon's sling. (Requires Combat 
+Expertise [Melee] and either 20 skill in Melee Combat or Unarmed Combat, or 
+class passive.)
 
 ==== MELEE WEAPONS ====
 
-=====	LEVEL 1 REQUIREMENT WEAPONS	=====
+=====LEVEL 1 REQUIREMENT WEAPONS=====
```
Comment (Turtlelord26) [inline CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt]: Mark

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt [hunk 5]
```diff
@@ -6,225 +6,287 @@ a 'Miss Chance'. This modifier is added to your combat dex modifier and
...
+
+==Eliazar Plasma Torch$240==
```
Comment (Turtlelord26) [inline CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt]: Mark

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt [hunk 6]
```diff
@@ -6,225 +6,287 @@ a 'Miss Chance'. This modifier is added to your combat dex modifier and
...
-Notes/Description: Plasma. Can quickly repair vehicles with welding materials, seal/cut through 
-	moderate doors, and make temporary cover. -40% stealth for being quiet on stealth attacks.
 
-----Gurbur G1 Combat Knife		$315	---- 
-Damage:		   40 + STR + 2*DEX + 1xBleed
-Range:     	   Adjacent (1m)
+	Plasma. Can quickly repair vehicles with welding materials, seal/cut through 
+moderate doors, and make temporary cover. -40% stealth for being quiet on stealth attacks.
+
+==Gurbur G1 Combat Knife$315==
```
Comment (Turtlelord26) [inline CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt]: Mark

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt [hunk 7]
```diff
@@ -6,225 +6,287 @@ a 'Miss Chance'. This modifier is added to your combat dex modifier and
...
+
+Damage:   40 + STR + 2*DEX + 1xBleed
+Range:        Adjacent (1m)
 Accuracy Modifier: +1
 DEX Requirement:   3
 
-----Gurber GS101 Combat Sword		$680	----
-Damage:		   50 + 2*STR + 2*DEX + 1xBleed
-Range:     	   2m
+==Gurber GS101 Combat Sword$680==
```
Comment (Turtlelord26) [inline CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt]: Mark

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt [hunk 8]
```diff
@@ -6,225 +6,287 @@ a 'Miss Chance'. This modifier is added to your combat dex modifier and
...
+
+Damage:   50 + 2*STR + 2*DEX + 1xBleed
+Range:        2m
 Accuracy Modifier: +2
 DEX Requirement:   4
 
-=====	LEVEL 5 REQUIREMENT WEAPONS	=====
+=====LEVEL 5 REQUIREMENT WEAPONS=====
```
Comment (Turtlelord26) [inline CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt]: Mark

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt [hunk 9]
```diff
@@ -6,225 +6,287 @@ a 'Miss Chance'. This modifier is added to your combat dex modifier and
...
+
+== Guri Grabber Knife$470==
```
Comment (Turtlelord26) [inline CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt]: Mark

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt [hunk 10]
```diff
@@ -6,225 +6,287 @@ a 'Miss Chance'. This modifier is added to your combat dex modifier and
...
-	Whenever you roll a successful grappling check, you may deal [3*STR or 3*DEX] damage to your opponent.
 
-----Guri Carbon Whip		$560	----
-Damage:		   30 + 3*STR + 2*DEX
-Range:     	   3m
+	You can grapple after any successful basic melee attack. +10% Skill on 
+grappling checks. Whenever you roll a successful grappling check, you may deal 
+[3*STR or 3*DEX] damage to your opponent.
+
+==Guri Carbon Whip$560==
```
Comment (Turtlelord26) [inline CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt]: Mark

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt [hunk 11]
```diff
@@ -6,225 +6,287 @@ a 'Miss Chance'. This modifier is added to your combat dex modifier and
...
 
-----SPECO SP10 VAE Combat Knife		$640	----
-Damage:		   45 + STR + 2*DEX + 1xBleed
-Range:     	   Adjacent (1m)
+	Requires "Weapon Use [Exotic]". Can grapple limbs from range (DEX based 
+grapple) and force movement (1m for every 5 [DEX or STR]). Can disarm weapons 
+from range according to Hand-to-Hand [trained] or [Expert] disarm rules (you 
+must have [Trained] or [Expert] to use their respective rules).
+
+==SPECO SP10 VAE Combat Knife$640==
```
Comment (Turtlelord26) [inline CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt]: Mark

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt [hunk 12]
```diff
@@ -6,225 +6,287 @@ a 'Miss Chance'. This modifier is added to your combat dex modifier and
...
 Accuracy Modifier: +1
 DEX Requirement:   4
-Notes/Description: Voltage Aligned Edge; ignores standard armor
 
-----SPECO SPC50 VAE Circular Saw		$695	----
-Damage:		   55 + STR + 1xBleed
-Range:     	   Adjacent (1m)
+	Voltage Aligned Edge; ignores standard armor
+
+==SPECO SPC50 VAE Circular Saw$695==
```
Comment (Turtlelord26) [inline CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt]: Mark

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt [hunk 13]
```diff
@@ -6,225 +6,287 @@ a 'Miss Chance'. This modifier is added to your combat dex modifier and
...
-	-50% on stealth rolls for being quiet during stealth attacks with this weapon.
 
-----SPECO SP30 VAE Combat Sword		$1245	----
-Damage:		   55 + 2*STR + 2*DEX + 1xBleed
-Range:     	   2m
+	Voltage Aligned Edge; ignores standard armor; can easily cut through most 
+materials. -50% on stealth rolls for being quiet during stealth attacks with 
+this weapon.
+
+==SPECO SP30 VAE Combat Sword$1245==
```
Comment (Turtlelord26) [inline CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt]: Mark

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt [hunk 14]
```diff
@@ -6,225 +6,287 @@ a 'Miss Chance'. This modifier is added to your combat dex modifier and
...
 DEX Requirement:   6
-Notes/Description: Voltage Aligned Edge; ignores standard armor
 
-=====	LEVEL 10 REQUIREMENT WEAPONS	=====
+	Voltage Aligned Edge; ignores standard armor
 
-----Eliazar E500 Assisted-Sledgehammer		$811	----
-Damage:		   50 + 6*STR
-Range:     	   2m
+=====LEVEL 10 REQUIREMENT WEAPONS=====
```
Comment (Turtlelord26) [inline CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt]: Mark

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt [hunk 15]
```diff
@@ -6,225 +6,287 @@ a 'Miss Chance'. This modifier is added to your combat dex modifier and
...
+
+==Eliazar E500 Assisted-Sledgehammer$811==
```
Comment (Turtlelord26) [inline CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt]: Mark

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt [hunk 16]
```diff
@@ -6,225 +6,287 @@ a 'Miss Chance'. This modifier is added to your combat dex modifier and
...
-Damage:		   50 + 5*STR
-Range:     	   Adjacent (1m)
+
+	2-handed weapon. Considered concussive (explosive) damage. Hits armor points 
+before physical damage if the target's armor has defense against explosive (i.e. 
+concussive) damage. If armor roll is greater than 50%, target rolls a DC 50% 
+fort save for a broken limb of your choice. -20% stealth rolls for being quiet 
+on stealth attacks. Reduces door-breaking DC by 30% per hit.
+
+==Eliazar E5500 Power Fist$880==
```
Comment (Turtlelord26) [inline CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt]: Mark

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt [hunk 17]
```diff
@@ -6,225 +6,287 @@ a 'Miss Chance'. This modifier is added to your combat dex modifier and
...
-----Muzashi VAE Katana		$1354	----
-Damage:		   60 + STR + 3*DEX + 2xBleed
-Range:     	   2m
+	Considered concussive (explosive) damage. Hits armor points before physical 
+damage if the target has defense against explosive (i.e. concussive) damage. If 
+armor roll is less than 50%, target rolls a DC 50% fort save for a being stunned 
+for 1 turn. -30% stealth rolls for being quiet on stealth attacks. Reduces 
+door-breaking DC by 45% per hit.
+
+==Muzashi VAE Katana$1354==
```
Comment (Turtlelord26) [inline CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt]: Mark

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt [hunk 18]
```diff
@@ -6,225 +6,287 @@ a 'Miss Chance'. This modifier is added to your combat dex modifier and
...
 Accuracy Modifier: +2
 DEX Requirement:   6
-Notes/Description: Requires "Weapon Use [Exotic]". Voltage Aligned Edge; ignores standard armor
 
-----Eliazar Plasma Sword		$987	----
-Damage:		   50 + STR + 2*DEX + 3xPlasma Burn
-Range:     	   2m
+	Requires "Weapon Use [Exotic]". Voltage Aligned Edge; ignores standard armor.
+
+==Eliazar Plasma Sword$987==
```
Comment (Turtlelord26) [inline CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt]: Mark

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt [hunk 19]
```diff
@@ -6,225 +6,287 @@ a 'Miss Chance'. This modifier is added to your combat dex modifier and
...
 
-----Gurber Throwing Knife	$80	----
-Damage:	15 + STR + DEX + [Thrown Weapon Skill]/2 + 1xBleed
-Ranged Miss Chance:
-	5 out to [DEX or STR/2] meters
-	6 out to 5 + [DEX or STR/2] meters
+	(When dividing Thrown Weapon Skill, drop the remainder)
+
+
+=====LEVEL 1 REQUIREMENT WEAPONS=====
```
Comment (Turtlelord26) [inline CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt]: Mark

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt [hunk 20]
```diff
@@ -6,225 +6,287 @@ a 'Miss Chance'. This modifier is added to your combat dex modifier and
...
-	6 out to 5 + STR + DEX/2 meters
-STR Requirement:   5
-AP Level:		3
-Notes/Description:	For melee, 40+2*STR+DEX+1xBleed, +1 melee accuracy. 1 per Secondary Ammo Slot.
+* 5 out to [DEX or STR/2] meters
+* 6 out to 5 + [DEX or STR/2] meters
+
+	For melee, 15+STR+2*DEX+1xBleed, +1 melee accuracy. 2 per Secondary Ammo Slot.
+
+=====LEVEL 5 REQUIREMENT WEAPONS=====
```
Comment (Turtlelord26) [inline CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt]: Mark

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt [hunk 21]
```diff
@@ -6,225 +6,287 @@ a 'Miss Chance'. This modifier is added to your combat dex modifier and
...
-Damage:	25 + STR + DEX + [Thrown Weapon Skill]/2 + 1xBleed
+	Ranged Miss Chance
+
+* 5 out to STR + DEX/2 - 1 meters
+* 6 out to 5 + STR + DEX/2 meters
+
+	For melee, 40 + 2 * STR + DEX + 1 x Bleed, +1 melee accuracy. 1 per 
+Secondary Ammo Slot.
+
+=====LEVEL 10 REQUIREMENT WEAPONS=====
```
Comment (Turtlelord26) [inline CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt]: Mark

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt [hunk 22]
```diff
@@ -233,79 +295,91 @@ behind the smoke and a target is on the complete opposite side, the smoke comple
...
-Notes/Description:	Each Target within the blast radius takes multiple hits. 45 damage per hit, automatically hits armor. 
+== ECSC Fragmentation Grenade $90 ==
+Damage:D10 burn damage to exact spot of impact
+Ranged Miss Chance (to land where intended):
+5 out to 10+[DEX or STR]*2 meters
+Armor Piercing Level:1
+
+	Each Target within the blast radius takes multiple hits. 45 damage per hit, automatically hits armor. 
 Targets take full damage to the torso, legs, and arms (if armor is pierced or not present): (3*45 damage)+2xBleed within a 2.5m radius.
 Only legs and torso (2*45 damage)+1xBleed within a 5m radius. At least one hit must damage health to cause  Bleed.
```
Comment (Turtlelord26) [inline CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt]: Line Length

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt [hunk 23]
```diff
@@ -233,79 +295,91 @@ behind the smoke and a target is on the complete opposite side, the smoke comple
...
+
+== RMM Frag+ Grenade $140 ==
+Damage:D10 burn damage to exact spot of impact
+Ranged Miss Chance (to land where intended):
+5 out to 10+[DEX or STR]*2 meters
+Armor Piercing Level:2
+
+	Each Target within the blast radius takes multiple hits. 50 damage per hit, automatically hits armor. 
 Targets take full damage to the torso, legs, and arms (if armor is pierced or not present): (3*50 damage)+2xBleed within a 3m radius.
 Only legs and torso (2*50 damage)+1xBleed within a 6m radius. At least one hit must damage health to cause Bleed.
```
Comment (Turtlelord26) [inline CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt]: Line Length

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt [hunk 24]
```diff
@@ -233,79 +295,91 @@ behind the smoke and a target is on the complete opposite side, the smoke comple
...
-Armor Piercing Level:	n/a		
-Notes/Description:	Creates a 7m radius cloud of burning, choking smoke. Targets take 30 Poison Damage per turn in the cloud.
+== Frontier Chemical Grenade $70 ==
+Damage:D10 burn damage to exact spot of impact
+Ranged Miss Chance (to land where intended):
+5 out to 10+[DEX or STR]*2 meters
+Armor Piercing Level:n/a
+
+	Creates a 7m radius cloud of burning, choking smoke. Targets take 30 Poison Damage per turn in the cloud.
 All Poison Damage is negated by use of a gas mask.
```
Comment (Turtlelord26) [inline CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt]: Line Length

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt [hunk 25]
```diff
@@ -233,79 +295,91 @@ behind the smoke and a target is on the complete opposite side, the smoke comple
...
-	5 out to 10+[DEX or STR]*2 meters	
-Armor Piercing Level:	6		
-Notes/Description:	Each Target within the blast radius takes multiple hits. 60 damage per hit, automatically hits armor. 
+== RMM HiEx Grenade $285 ==
+Damage:D10 burn damage to exact spot of impact
+Ranged Miss Chance (to land where intended):
+5 out to 10+[DEX or STR]*2 meters
+Armor Piercing Level:n/a
+
+	2x inner blast radius when indoors. 120 explosive damage within a 2m radius, 60 explosive within a 5m radius.
```
Comment (Turtlelord26) [inline CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt]: Line Length

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt [hunk 26]
```diff
@@ -233,79 +295,91 @@ behind the smoke and a target is on the complete opposite side, the smoke comple
...
+== Frontier Plasma Grenade FP-G5 $360 ==
+Damage:D10 burn damage to exact spot of impact
+Ranged Miss Chance (to land where intended):
+5 out to 10+[DEX or STR]*2 meters
+Armor Piercing Level:6
+
+	Each Target within the blast radius takes multiple hits. 60 damage per hit, automatically hits armor. 
 Targets take full damage to the torso, legs, and arms (if armor is pierced or not present): (3*60 damage)+3xPlasma Burn within a 3m radius.
-Only legs and torso (2*60 damage)+1xPlasma Burn within a 5m radius. At least one hit must damage health to cause Plasma Burn.		
+Only legs and torso (2*60 damage)+1xPlasma Burn within a 5m radius. At least one hit must damage health to cause Plasma Burn.
```
Comment (Turtlelord26) [inline CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt]: Line Length

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt [hunk 27]
```diff
@@ -233,79 +295,91 @@ behind the smoke and a target is on the complete opposite side, the smoke comple
...
-Armor Piercing Level:	0		
-Notes/Description:	Everyone in the room gets blinded for 2 turns. Once per turn, You can 
+== RMM Flash-Bang Distraction and Clearing Device $50 ==
+Damage:D10 burn damage to exact spot of impact
+Ranged Miss Chance (to land where intended):
+6 out to 10+[DEX or STR]*2 meters
+Armor Piercing Level:0
+
+	Everyone in the room gets blinded for 2 turns. Once per turn, You can 
 attempt to break the blindness with a perception check at 85% (disregarding bonuses). If the grenade
```
Comment (Turtlelord26) [inline CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt]: Line Length

**Review by Turtlelord26 — CHANGES_REQUESTED** [review]
Remaining spacing issues marked. Remaining line length issues noted.

**Comment by bleehu** [comment]
Ouch. Cut me bleeding. Red all over. 

How's that now?

**Comment by Turtlelord26** [comment]
Much better but I'm feeling perfectionistish at the moment. Watch this space.

**Review by Turtlelord26 — APPROVED** [review]

**Pull request merged**